dashboards/key_metrics/app.py

```diff
@@ -35,10 +35,11 @@ def load_api():
 lp = st.Page("views/lp.py", title="Liquidity Providers")
 perps = st.Page("views/perps.py", title="Perps")
 token = st.Page("views/token.py", title="$SNX Token")
+accounts = st.Page("views/accounts.py", title="Accounts Activity")
 
 # navigation
 pages = {
-    "": [cross_chain, lp, perps, token],
```

Comment:
lets keep the view code but remove the additional dashboard for now, until we write models for these